videogame-tiled.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * videogame-tiled.js
+ *
+ * This extension adds a 'videogame map format' (Ctrl+F) type to the Export As
+ * menu, which can be used to generate matrix maps as used by the Scene.build()
+ * method from the videogame library (https://github.com/diogoeichert/videogame)
+ */
+
+tiled.registerMapFormat("videogame", {
+    name: "videogame map format",
+    extension: "json",
+
+    write: (map, fileName) => {
+        for (let i = 0; i < map.layerCount; ++i) {
+            const layer = map.layerAt(i);
+
+			if (!layer.isTileLayer) {
```

Comment:
Please avoid using a mix of tabs and spaces. In the other examples I used 4-space indentation, no tabs.

Reply:
Thanks for pointing that out, I've got used to having a linter in the project and no longer worry about that. If you're interested, I can create a separate pull request adding eslint definitions to the project that help enforce the coding style, it integrates nicely with editors such as VS Code. I'll just create it and then you can judge if it's a benefit or not.

Reply:
Ah, that may be a good idea, thanks!